build/jenkins/Jenkinsfile.deployment

```diff
@@ -70,7 +70,7 @@ pipeline {
     stage("Monitor Deployment") {
       steps {
         script {
-          echo "to do"
+          sh "make k8s-check-deployment-of-replica-sets"
```

Comment:
That's the first part of the ticket, the second part is to make sure that we can reach the API via the R53 name (that's the path that the smoke tests use).  Basically, when spinning an environment up from scratch, it can take up to 5 mins before it can be accessed via the R53 name.  So, we now need an additional step here to keep polling the R53 name until it is able to connect to the API via that route, otherwise the smoke tests will fail as they won't be able to connect. Hope that makes sense.